torchvision/prototype/transforms/_geometry.py

```diff
@@ -15,6 +15,9 @@
 from ._utils import _check_sequence_input, _setup_angle, _setup_size, has_all, has_any, query_bounding_box, query_chw
 
 
+DType = Union[torch.Tensor, PIL.Image.Image, features._Feature]
```

Comment:
Seeing that this is defined in every module, can we maybe put this in `_utils`?

Reply:
Yeah fine by me.